<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pending_book_req</class>
 <widget class="QMainWindow" name="pending_book_req">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Requests</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="acceptreq_button">
         <property name="text">
          <string>Accept</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rejectreq_button">
         <property name="text">
          <string>Reject</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="returnedbook_button">
         <property name="text">
          <string>Returned</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>acceptreq_button</tabstop>
  <tabstop>rejectreq_button</tabstop>
  <tabstop>returnedbook_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
